ls1intum/Artemis
# [Followup] Student-exam-detail rest call checks and logs

### Checklist

### Motivation and Context


The REST calls in the student-exam-detail that change the submitted state of the individual student-exam did not check if the individual working time was already over. I added this, and put the logs at the end, so they are only executed after all checks pass.

### Steps for Testing


Create an exam, for one student increase the individual working time. Now while the exam has already started, and the normal end date of the exam ran out, go to the student-exam-detail again, and try to use the "change student exam to submitted" button. 
There should be an error message:
<img src="https://user-images.githubusercontent.com/33342534/109386254-c4ddeb80-78f9-11eb-9965-6d36b6b520e0.png" width="400">
